### ISSUE TITLE
KDE file-picker (portal) does not allow access to app-files, making plug-in selection/loading impossible

### ISSUE THREAD

**Issue by cobratbq** [flathub/org.claws_mail.Claws-Mail#34]
The KDE file-picker does not offer access to the flat-packaged app files, therefore it is impossible to select plug-ins to load.

At this time, it is not clear to me how to resolve this. Most likely, it is impossible or there is some expression for the manifest, e.g. to enable/disable something or some start-up parameter.

**Comment by scabala** [comment]
I did some investigation on the issue and it seems that there are few options how to handle that:
 - path Claws Mail to use older `GtkFileChooserDialog` just for loading plugins - no idea if that works
 - try 'path rewriting' (whatever that might be) as per [this](https://github.com/flatpak/flatpak/issues/5846#issuecomment-2186084490) comment
 - contact Claws Mail developers about ideas how to handle that
 - let users handle that in some way
 - do not upgrade to 4.2.0 and above

I will try to focus on first 2 points, hopefully with some spare time on my hands I will be able to achieve something.

**Comment by cobratbq** [comment]
> I did some investigation on the issue and it seems that there are few options how to handle that:
> 
> * path Claws Mail to use older `GtkFileChooserDialog` just for loading plugins - no idea if that works
> * try 'path rewriting' (whatever that might be) as per [this](https://github.com/flatpak/flatpak/issues/5846#issuecomment-2186084490) comment
> * contact Claws Mail developers about ideas how to handle that
> * let users handle that in some way
> * do not upgrade to 4.2.0 and above

* Keep with (updated) version 3.x, as this does not use the GTK-3 toolkit which makes this distinction with the file-picker.

That's the solution I was going to go with, as long as there were no better solutions, until #40 builds started failing for yet other reasons.

**Comment by scabala** [comment]
Yes, that's true, forgot about this one. I just wanted to stick with 4.x series and I think I found solution. PR will follow up shortly.

**Comment by cobratbq** [comment]
Solved by proposal in #42.
